### PR TITLE
Add slim syntax highlighting

### PR DESCRIPTION
**Pull request by Antti**
Add [Slim](http://slim-lang.com/) syntax highlighting via [Slim extension](https://marketplace.visualstudio.com/items?itemName=sianglim.slim)

### CODE REVIEW

**Review by reneklacan — APPROVED** [review]
looks great :heart:

**Comment by majtymit** [comment]
lovely 👍

**Review by octref — APPROVED** [review]
Good work, nothing need to change. Thank you 👍

File: syntaxes/vue-generated.json [hunk 1]
```diff
@@ -6,6 +6,49 @@
   ],
   "uuid": "5512c10d-4cc5-434c-b8fc-53b912f55ab3",
   "patterns": [
+    {
```
Comment (yoyo930021) [inline syntaxes/vue-generated.json:9]: @Antti
I notice this block code.
it isn't generated from yaml.

Is this code necessary?

Comment (Antti) [reply syntaxes/vue-generated.json:9]: @yoyo930021  I have only edited the YAML file and run the generator, so I believe it was generated from it. At least at that point of time.

Comment (yoyo930021) [reply syntaxes/vue-generated.json:9]: That's weird.
Now when running will disappear.
I don't think it should be necessary and I will remove it.

**Pull request merged**